app/uk/gov/hmrc/gform/testonly/TestOnlyController.scala

```diff
@@ -625,10 +674,10 @@ class TestOnlyController(
     val tableRows: List[List[TableRow]] = snapshots.map { snapshot =>
       List(
         TableRow(
-          content = HtmlContent(Html(snapshot.templateId))
+          content = HtmlContent(Html(snapshot.templateId.value))
```

Comment:
I wonder if we shouldn't display original template id (without prefix)?

Reply:
I think to dedicate the full column for it is a waste of space. May be in the snapshot page somewhere?

Reply:
I meant instead of this column. Who will ever need to see any random prefix here?

Reply:
ok, I will change it to the original template